dubbo-common/src/main/java/org/apache/dubbo/common/config/ModuleEnvironment.java

```diff
@@ -93,9 +93,6 @@ public List<Map<String, String>> getConfigurationMaps(AbstractConfig config, Str
 
     @Override
     public Configuration getDynamicGlobalConfiguration() {
-        if (dynamicConfiguration == null) {
-            return applicationDelegate.getDynamicGlobalConfiguration();
-        }
```

Comment:
Should support use applicationModel's dynamic configuration if not set to ModelEnvironment

Reply:
Please refer to the code following these two lines of code ：

```
 if (dynamicConfiguration == null) {
                if (logger.isWarnEnabled()) {
                    logger.warn(
                            COMMON_UNEXPECTED_EXCEPTION,
                            "",
                            "",
                            "dynamicConfiguration is null , return globalConfiguration.");
                }
                return getConfiguration();
 }
```

if dynamicConfiguration  is null, it will use the result of method `getConfiguration()`

```
    @Override
    public CompositeConfiguration getConfiguration() {
        if (globalConfiguration == null) {
            CompositeConfiguration configuration = new CompositeConfiguration();
            configuration.addConfiguration(applicationDelegate.getConfiguration());
            configuration.addConfiguration(orderedPropertiesConfiguration);
            globalConfiguration = configuration;
        }
        return globalConfiguration;
    }
```
method `getConfiguration` will return applicationModel's dynamic configuration  and **ordered properties configuration**